dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/export/event/EventOperationParamsMapperTest.java

```diff
@@ -140,12 +138,10 @@ public void setUp() {
             createOrgUnit("captureScopeChild", "captureScopeChildUid"),
             createOrgUnit("searchScopeChild", "searchScopeChildUid")));
 
-    user = new User();
+    User user = new User();
     user.setUsername("test");
     user.setOrganisationUnits(Set.of(orgUnit));
-
-    injectSecurityContext(UserDetails.fromUser(user));
-    when(userService.getUserByUsername(anyString())).thenReturn(user);
+    currentUser = UserDetails.fromUser(user);
```

Comment:
I would stick to `currentUser` only in code that knows it came from the `CurrentUserUtil`. The code getting the user in does not know that so I would name it `user`.

Reply:
Done